cmd/embedded-cluster/flags.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/replicatedhq/embedded-cluster/pkg/defaults"
+	"github.com/urfave/cli/v2"
+	k8snet "k8s.io/utils/net"
+)
+
+func getAdminColsolePortFlag() cli.Flag {
+	return &cli.StringFlag{
+		Name:   "admin-console-port",
+		Usage:  "Port on which the Admin Console will be served",
+		Value:  strconv.Itoa(defaults.AdminConsolePort),
```

Comment:
Nice. Came here looking for this because I like the default value to show in the help menu so users can more easily determine if they need to set it to something else.

Reply:
Is there a range of ports that that are valid that we should validate this against? Maybe 80-32767 or whatever the node port range is set to?

Reply:
Oh I see your TODO down lower